test-resources/go/feature_flag/system_1/const_same_file/input/sample.go

```diff
@@ -87,6 +87,19 @@ func callerFunc() {
 
 // should not replace the function name
 func isFlagEnabledFunc() bool {
+    // Should not delete this comment and the func below
+    t.Run("message", func(t *Foobar.T) {
+        isFlgEnabled := exp.BoolValue(staleFlagConst)
+        fmt.Println("some logging statement")
+    })
+    
+    // Should not delete this comment and the func below
+    t.Run1(func(t *Foobar.T) {
+        isFlgEnabled1 := exp.BoolValue(staleFlagConst)
+        fmt.Println("some other logging statement")
+    }, "other_message" )
```

Comment:
In both these examples, the comma is before the deleted node, but the change handles both before and after, right? Any test case case you can think of for the "after" case?

Reply:
aah I see ur point. I actually had a test case like that, but I had it slightly wrong. I reordered the statements here. 
In our earlier logic we would delete `},` in addition to `isFlgEnabled1 := exp.BoolValue(staleFlagConst)` 

But now we don't, because we see that there is a node which ends between the ending of deleted node and comma

Reply:
Ok, but this change makes the comment above outdated, not?

```
// Should not delete this comment and the func below
```

It should now say something about the trailing `},`

Reply:
good catch. Addressed.